hw/top_earlgrey/rdc/rdc_waivers.spi_device.tcl

```diff
@@ -16,3 +16,15 @@ set_rule_status -rule {W_RST_COMBO_LOGIC} -status {Waived}  \
 set_rule_status -rule {W_RST_COMBO_LOGIC} -status {Waived} \
   -expression {(SourceReset=~"*.u_spi_device.rst_*_buf")}  \
   -comment {rst csb, tpm_csb are reset merging with scan chain.}
+
+
+# RDC could not catch the relation between CSb and SCK
+# However, if CSb mis-behaved, the logic may/maynot push entries to CMDADDR
+# FIFO and WR FIFO.
+set_rule_status -rule {E_RST_METASTABILITY} -status {Waived} \
+  -expression {(MetaStableFlop=~"*.u_spi_tpm.u_cmdaddr_buffer.fifo_wptr*")} \
+  -comment {when csb de-asserted, SCK should be quiscent. If not, the behavior is unpredictable}
```

Comment:
quiscent -> quiescent

Reply:
just to quickly check, is this covered by design already? Because when we reset the spi_device block via rstmgr, that also resets the mode to "not" be `tpm` right? Does that imply csb and sck will be stable? I know we had looked at adding CG on the sck when a mode is not selected...just can't remember if it's there.

Reply:
As far as I know, no. The design does not block the CSb, TPM when IP is being reset.
 
The waiver above is when TPM CSb is being asserted without waiting more than half SCK period (violating the protocol). In that case, the premature FIFO wvalid signal may put an entry to the FIFO (`wvalid` is comb. logic comparing `bitcnt` with `'h 1F`)

Reply:
i'm sorry, i have no idea why i thought IP reset was involved. 
Do we know for sure though that SCK has to be quiescent by spec? Or is it just that timing of SCK should not conflict with CSB?

Reply:
Yep TPM spec describes the timing (CS# may de-assert minimum 5ns after SCK negedge)

Reply:
ah sounds good! Could you just put that in the comment?

Reply:
Yep. Added! Thanks for the tip.